src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseFileServiceTests.cs

```diff
@@ -1467,23 +1472,56 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
                 Assert.Equal("93 Kb", fileInfo.Size);
                 Assert.Equal(Ancillary, fileInfo.Type);
             }
+
             MockUtils.VerifyAllMocks(blobStorageService);
         }
 
         [Fact]
-        public async Task GetFile_NoReleaseFile()
+        public async Task GetFile_NoRelease()
         {
             var blobStorageService = new Mock<IBlobStorageService>(MockBehavior.Strict);
 
             await using (var contentDbContext = InMemoryApplicationDbContext())
             {
-                var service = SetupReleaseFileService(contentDbContext: contentDbContext,
+                var service = SetupReleaseFileService(
+                    contentDbContext: contentDbContext,
                     blobStorageService: blobStorageService.Object);
 
                 var result = await service.GetFile(Guid.NewGuid(), Guid.NewGuid());
+
+                Assert.True(result.IsLeft);
+                Assert.IsType<NotFoundResult>(result.Left);
```

Comment:
Can now use `result.AssertNotFound()` which'll do these 2 checks in one.

Reply:
Done!